### PR TITLE
Homework3: Sergey Streltsov

### PR DESCRIPTION
**Pull request by Mantikor**

### CODE REVIEW

File: src/homework03.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    """
+    return {n: n * n for n in range(1, number + 1)}
+
+
+def count_characters(count_me_string):
+    """Dictionary practice
+
+    """
+    return {ch: count_me_string.count(ch) for ch in count_me_string}
```
Comment (belovmd) [inline src/homework03.py]: it has O(n**2) complexity. 
There is O(n) solution - please take a look at dicts methods

File: src/homework03.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+def list_practice():
+    """List practice
+
+    """
+    src_lst = [i + j for i in 'ab' for j in 'bcd']
+    print(src_lst)
+    new_lst = src_lst[::2]
+    print(new_lst)
+    sec_lst = [str(i) + 'a' for i in range(1, 5)]
```
Comment (belovmd) [inline src/homework03.py]: there is more native way instead of range

**Review by belovmd — COMMENTED** [review]
1. Please add tasks description
2. Please address these minor issues:

**Comment by Mantikor** [comment]
Fixes for all remarks

**Review by belovmd — COMMENTED** [review]
8.5
it's maximal score for full completion of this hw.

**Pull request closed without merging**